cmd/create/cluster/cmd.go

```diff
@@ -2267,10 +2267,13 @@ func run(cmd *cobra.Command, _ []string) {
 	var subnets []ec2types.Subnet
 	mapSubnetIDToSubnet := make(map[string]aws.Subnet)
 	if useExistingVPC || subnetsProvided {
-		initialSubnets, err := getInitialValidSubnets(awsClient, subnetIDs, r.Reporter)
-		if err != nil {
-			_ = r.Reporter.Errorf("Failed to get the list of subnets: %s", err)
-			os.Exit(1)
+		var initialSubnets []ec2types.Subnet
+		if len(subnetIDs) > 0 || subnetsProvided {
```

Comment:
I think we can change this to simplify, is there a specific reason we need `if len(subnetIDs) > 0`?

`subnetsProvided` is already the result of `len(subnetIDs) > 0`, so we can just use `subnetsProvided` instead

Beyond this, the pre-existing if statement on line 2269 already has `subnetsProvided`

So.. I think we can remove the new if statement, and the new `var initialSubnets []ec2types.Subnet` definition, since after removing this new if statement, the definition on 2272 will be in the proper scope again